src/screens/FeedScreen/RenderList.js

```diff
@@ -161,6 +165,7 @@ const RenderListFeed = (props) => {
             topics={item?.topics}
             item={item}
             onNewPollFetched={onNewPollFetched}
+            hasComment={hasComment}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `hasComment` prop is being passed to the `Content` component. Ensure that this prop is used correctly in the `Content` component and that it doesn't conflict with any other props with the same name.
```diff
+ hasComment={hasComment}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->